arthur/exts/directory/ldap.py

```diff
@@ -123,16 +123,18 @@ async def generate_creds(self, interaction: discord.Interaction, _button: ui.But
             )
             return
 
-        bootstrap_type, password = await self.cog.bootstrap(user)
+        bootstrap_type, password, uid = await self.cog.bootstrap(user)
 
         if bootstrap_type == BootstrapType.CREATION:
             title = "Account Creation"
             logger.info(f"Created account for {user}")
         else:
             title = "Password Reset"
-            logger.info(f"Reset password for {user}")
+            logger.info(f"Reset password for {user} with {uid}")
```

Comment:
Looks good! All I would say is we should be a bit more clear in this log line on what "with XXX" means.


```suggestion
            logger.info(f"Reset password for {user} with user ID: {uid}")
```

Reply:
Ahhh, embarrassing. I had it as `...with uid: {uid}"`, but I had to fork and reclone the repo since pushing my changes failed due to lack of write permission, and I fudged this.

Reply:
Addressed in force push: https://github.com/python-discord/king-arthur/compare/394430962682072889564ef2fe3e49b93675b809..a8502dfc7de28a5b676a744f7323e177a88c51e0